src/actions/ProfileActions.js

```diff
@@ -1,42 +1,31 @@
 import AsyncActionType from './AsyncActionType';
 
-export const FIELD_OPEN = 'FIELD_OPEN';
-export const FIELD_CLOSE = 'FIELD_CLOSE';
 export const FETCH_PROFILE = new AsyncActionType('PROFILE', 'FETCH_PROFILE');
 export const SAVE_PROFILE = new AsyncActionType('PROFILE', 'SAVE_PROFILE');
 export const SAVE_PROFILE_PHOTO = new AsyncActionType('PROFILE', 'SAVE_PROFILE_PHOTO');
 export const DELETE_PROFILE_PHOTO = new AsyncActionType('PROFILE', 'DELETE_PROFILE_PHOTO');
-export const UPDATE_DRAFTS = 'UPDATE_DRAFTS';
-export const RECEIVE_PREFERENCES = 'RECEIVE_PREFERENCES';
+export const OPEN_FORM = 'OPEN_FORM';
+export const CLOSE_FORM = 'CLOSE_FORM';
+export const UPDATE_ACCOUNT_DRAFT = 'UPDATE_ACCOUNT_DRAFT';
+export const UPDATE_VISIBILITY_DRAFT = 'UPDATE_VISIBILITY_DRAFT';
+export const RESET_DRAFTS = 'RESET_DRAFTS';
 
+// FETCH PROFILE ACTIONS
 
-export const openField = fieldName => ({
-  type: FIELD_OPEN,
-  fieldName,
-});
-
-export const closeField = fieldName => ({
-  type: FIELD_CLOSE,
-  fieldName,
-});
-
-export const updateDrafts = drafts => ({
-  type: UPDATE_DRAFTS,
-  drafts,
+export const fetchProfile = username => ({
+  type: FETCH_PROFILE.BASE,
+  payload: { username },
 });
 
 export const fetchProfileBegin = () => ({
   type: FETCH_PROFILE.BEGIN,
 });
 
-export const fetchProfileSuccess = profile => ({
+export const fetchProfileSuccess = (account, preferences, certificates) => ({
```

Comment:
fetchProfileSuccess dumps all our fancy new data into the reducer now - not just the misnomered "profile" (which is really "account" data)